Tests/js/views/assets/ez-contenteditview-tests.js

```diff
@@ -526,7 +529,14 @@ YUI.add('ez-contenteditview-tests', function (Y) {
                 method: 'isValid',
                 returns: this.valid
             });
-
+            Y.Mock.expect(this.formView, {
+                method: 'setServerSideErrors',
+                args: [Y.Mock.Value.Object],
```

Comment:
it actually expects an Array according to https://github.com/ezsystems/PlatformUIBundle/pull/672/files#diff-85f756e5700996ae597d2175af6214cdR185